daemon/daemon_unix.go

```diff
@@ -550,7 +550,7 @@ func verifyPlatformContainerSettings(daemon *Daemon, hostConfig *containertypes.
 	// check for various conflicting options with user namespaces
 	if daemon.configStore.RemappedRoot != "" && hostConfig.UsernsMode.IsPrivate() {
 		if hostConfig.Privileged {
-			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces")
+			return warnings, fmt.Errorf("Privileged mode is incompatible with user namespaces.  To temporarily disable user namespaces, use '--userns=host'.")
```

Comment:
We recently rewrote these messages to avoid referring to command-line flags, given that this error is returned by the daemon, so any client of the API could be in use.